components/player-card/player-recent-matches.tsx

```diff
@@ -20,6 +21,25 @@ const PlayerRecentMatches = ({
   playerMatchesData: Array<any>;
   error: string;
 }) => {
+  const [sortStatus, setSortStatus] = React.useState<DataTableSortStatus>({
+    columnAccessor: "Played",
+    direction: "asc",
+  });
+  const [sortedData, setSortedData] = React.useState(sortBy(playerMatchesData, "Played"));
```

Comment:
I think we need to add type to the sorted data, this should do the trick
```
React.useState<Array<any>>(sortBy(playerMatchesData, "Played"));
```
IT would be nice to have proper type but you can proceed with any in this case 👍